flow-typed/npm/fetch-jsonp_vx.x.x.js

```diff
@@ -1,32 +1,19 @@
-// flow-typed signature: 40fcfe46cbedcd0342c9ee88f6c6aa21
-// flow-typed version: <<STUB>>/fetch-jsonp_v^1.0.2/flow_v0.40.0
-
-/**
- * This is an autogenerated libdef stub for:
- *
- *   'fetch-jsonp'
- *
- * Fill this stub out by replacing all the `any` types.
- *
- * Once filled out, we encourage you to share your work with the 
- * community by sending a pull request to: 
- * https://github.com/flowtype/flow-typed
- */
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// adapted from https://github.com/camsong/fetch-jsonp/blob/master/index.d.ts
 declare module 'fetch-jsonp' {
```

Comment:
I'm curious, (but not enough to file an issue) is it useful to have these stubbed out flow-typed files, or should we remove them? They were auto generated by the flow-typed CLI, but aren't actual definitions.

Reply:
I think this is only useful to generate a skeleton to fill in when writing a real libdef, but otherwise brings no value.